src/test/groovy/org/mockito/release/internal/gradle/AutoVersioningPluginTest.groovy

```diff
@@ -0,0 +1,15 @@
+package org.mockito.release.internal.gradle
+
+import org.gradle.testfixtures.ProjectBuilder
+import spock.lang.Specification
+
+class AutoVersioningPluginTest extends Specification {
```

Comment:
Thank you for adding a lifecycle test. I often forget to do so for the plugins I create!